tests/foreman/cli/test_user.py

```diff
@@ -9,21 +9,59 @@
 
 """
 
-from ddt import ddt
 from fauxfactory import gen_alphanumeric, gen_string
 from robottelo.cli.base import CLIReturnCodeError
-from robottelo.cli.factory import (
-    make_location,
-    make_org,
-    make_role,
-    make_user,
-)
+from robottelo.cli.factory import make_location, make_org, make_role, make_user
 from robottelo.cli.user import User as UserObj
-from robottelo.decorators import data, stubbed, skip_if_bug_open
+from robottelo.decorators import stubbed, skip_if_bug_open
 from robottelo.test import CLITestCase
 
 
-@ddt
+def valid_data_user():
+    """Returns a list of valid data for user tests.
+
+    Note: utf8 data is being rejected by the satellite server so it is not
```

Comment:
I am surprised why the general cli tests run in smoke test mode, this really needs investigation.  But it is beyond the scope of this PR.